sources/packages/api/src/esdc-integration/e-cert-integration/models/e-cert-integration-model.ts

```diff
@@ -36,3 +43,27 @@ export type Award = Pick<
   DisbursementValue,
   "valueType" | "valueCode" | "valueAmount"
 >;
+
+/**
+ * Codes used to start all the lines of the e-Cert
+ * files sent to ESDC.
+ */
+export enum RecordTypeCodes {
```

Comment:
Is it safe to share the same enum between PartTime and FullTime?

Reply:
I think its fine, as long as we have the variables declared in it are different. I did this as the declaration variables in the base class can be reused.

Reply:
I still think that these are areas that we talked about not "merging" and keeping them segregated would be better for maintenance in the future. Open for discussion.